instances/UBERONParcellation/ansiformLobuleCrusII.jsonld

```diff
@@ -0,0 +1,25 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/UBERONParcellation/ansiformLobuleCrusII",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/UBERONParcellation",
+    "description": "",

```

Comment:
left over (I think this issue got lost in all my comments...): please set this to `"description": null,`

Reply:
Done.